lambdas/utils/aws/s3.py

```diff
@@ -150,14 +115,71 @@ def download_object_byte_range(
         resp = s3_client.get_object(Bucket=bucket, Key=key, Range=range_header)
         return resp["Body"].read()
 
+    @staticmethod
+    def _decode_base64_sha256(base64_checksum: str | bytes) -> str:
+        binary_checksum = base64.b64decode(base64_checksum)
+        return binascii.hexlify(binary_checksum).decode("ascii")
+
     @classmethod
-    def calculate_checksum_for_large_object(
+    def get_object_checksum(
```

Comment:
Now, a single entrypoint for the `AIP` class to get a checksum for a single S3 object.